balancer/balancer.go

```diff
@@ -196,14 +204,32 @@ func (b *BalancerImpl) getMistLoadBalancerServers(ctx context.Context) (map[stri
 		return nil, err
 	}
 
-	var mistResponse map[string]interface{}
+	var mistResponse map[string]any
 
 	err = json.Unmarshal([]byte(string(bytes)), &mistResponse)
 	if err != nil {
 		return nil, err
 	}
 
-	return mistResponse, nil
+	output := make(map[string]bool, len(mistResponse))
```

Comment:
not sure why do we need a map? Will it ever contain `false` as a value?

Reply:
Nope! But Go doesn't have a set, so this is the idiomatic way to do it. Actually, doing a quick Google, it looks like `map[string]struct{}` [takes zero bytes per entry instead of one byte per entry](https://stackoverflow.com/questions/34018908/golang-why-dont-we-have-a-set-datastructure#comment89711306_34020023), so I'll do that instead.

Reply:
I wasn't sure if we need a set when trusting mist not returning duplicates